datahub-web/@datahub/metadata-types/addon/constants/common/fabric-type.ts

```diff
@@ -12,17 +12,5 @@ export enum FabricType {
   // Designates production fabrics, such as prod-ltx1, prod-lva1 etc.
```

Comment:
Can we remove examples as well like `prod-ltx1` which seems to be LinkedIn internal details?